### ISSUE TITLE
_open and _close index not supported in AWS service

### ISSUE THREAD

**Issue by CosX** [Epinova/Epinova.Elasticsearch#45]
This is an issue experienced with the implementation of synonyms. Today the list of synonyms closes an index, loads settings including synonym list and reopens index.
 
Comments on https://stackoverflow.com/a/33923322/1702945 suggests creating an alias and rebuilding index on changed settings.

**Comment by lonnkvist** [comment]
There are plans to use the alias approach in other areas, to avoid downtime when doing changes in the settings. Unfortunately there is no ETA on this yet.

A pre-release supporting synonyms_path exists, but that might not help in your scenario?

**Comment by CosX** [comment]
Does the pre-release of synonyms_path avoid using the _close endpoint?

**Comment by lonnkvist** [comment]
Yes, this is just a flat file in the config-folder. It doesn't touch the settings. The downside is that it is harder to maintain, espesially from an editors perspective.
